src/Mod/Draft/draftguitools/gui_base.py

```diff
@@ -214,3 +221,134 @@ def commit(self, name, func):
             that will be executed.
         """
         self.commit_list.append((name, func))
+
+
+class PolarCircularBase(GuiCommandBase):
```

Comment:
Don't forget adding a docstring explaning what this class does and what it is intended for. In this case, you would write something like, "this is meant to be subclassed by Polar and Circular to allow us select an edge in the interface".